src/main/java/com/github/wovnio/wovnjava/Interceptor.java

```diff
@@ -29,6 +31,9 @@ private String apiTranslate(String lang, String body) {
             String translatedBody = api.translate(lang, convertedBody);
             responseHeaders.setApiStatus("Success");
             return converter.restore(translatedBody);
+        } catch (ApiNoPageDataException e) {
+            responseHeaders.setApiStatus(Interceptor.NO_API_DATA_STATUS);
```

Comment:
How about defining an error name at `ApiNoPageDataException.java` and writing it like `e.getName()` to get it?
I don't think `Interceptor.java` needs to know the name of the error.

Reply:
That's also good. Let me implement that.